includes/documents/class-wcpdf-invoice.php

```diff
@@ -506,6 +506,38 @@ public function get_pdf_settings_fields( $option_name ) {
 					'description'	=> sprintf(__( "Disable document when the order total is %s", 'woocommerce-pdf-invoices-packing-slips' ), function_exists('wc_price') ? wc_price( 0 ) : 0 ),
 				)
 			),
+			array(
```

Comment:
Maybe this setting would be better placed below the document date?

Reply:
Do you mean below the "Display invoice date"?

Reply:
Yes. What you think?

Reply:
Sure, it seems better.